org.kde.kweather.json

```diff
@@ -0,0 +1,38 @@
+{
+    "id": "org.kde.kweather",
+    "branch": "master",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "kweather",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--system-talk-name=org.freedesktop.GeoClue2"
+    ],
+    "separate-locales": false,
+
+    "modules": [
+        {
+            "name": "kweathercore",
+            "config-opts": [ "-DBUILD_TESTING=OFF" ],
+            "buildsystem": "cmake-ninja",
+            "sources": [ { "type": "git", "url": "https://invent.kde.org/libraries/kweathercore.git", "branch": "release/21.05", "commit": "97dc22c6d1bc32f1e946c420dd0c102f065e46ed" } ]
```

Comment:
You should omit the branch. Because as soon as there are more commits on the branch, it will fail since branch HEAD and commit won't match. This applies to the other modules too.